Makefile

```
@@ -11,7 +11,7 @@ install:
 	go install github.com/pulumi/upgrade-provider
 
 lint:
-	golangci-lint run
+	golangci-lint run -E gci --fix
```

My prior is that `make lint` shouldn't make code changes. Instead changes should be delegated to another command, such as `make lint.fix`.

`-E` -> `--enable` because I didn't know what `-E` meant, but `--enable` is readable.

```suggestion
	golangci-lint run --enable gci
```

---

Yeah, that's totally fair - I think a good state is where we can run `make lint` and see necessary changes.